doc/crypto/api.db/psa/crypto.h

```diff
@@ -24,6 +24,9 @@ typedef uint32_t psa_pake_primitive_t;
 typedef uint8_t psa_pake_primitive_type_t;
 typedef uint8_t psa_pake_role_t;
 typedef uint8_t psa_pake_step_t;
+typedef struct psa_key_custom_production_t {
+    uint32_t flags;
+} psa_key_custom_production_t;
```

Comment:
Now that I'm typing it out, I feel the type name doesn't quite work: this data type doesn't exactly describe a “custom production”, but parameters for a custom production. The name makes me think of an operation context rather than configuration data. So `psa_key_custom_parameters_t` feels more natural, or maybe even `psa_custom_key_production_parameters_t` (I also feel that the `key` part is in a bit of a weird place). But maybe I'm overdoing it with the long descriptive name?

Reply:
I kind-of thought so too as I worked on the tweaks. My inclination would be to go for `psa_custom_key_parameters_t`, rather than more precise-but-long `psa_custom_key_production_parameters_t`. Key parameters/attributes are primarily used for key creation in the API, so I think we can get away without 'production' in the name?

Reply:
Agreed. I've updated https://github.com/Mbed-TLS/mbedtls/pull/9235 to use `psa_custom_key_parameters_t`.

Reply:
Thanks, I've updated the PR with this identifier name.